app/main/views/frameworks.py

```diff
@@ -11,3 +12,18 @@ def list_frameworks():
     return jsonify(
         frameworks=[f.serialize() for f in frameworks]
     )
+
+
+@main.route('/frameworks/g-cloud-7/stats', methods=['GET'])
+def get_framework_stats():
+
+    seven_days_ago = datetime.datetime.now() + datetime.timedelta(-7)
+
+    return str({
+        'drafts': DraftService.query.filter(DraftService.status == "not-submitted").count(),
+        'complete': DraftService.query.filter(DraftService.status == "submitted").count(),
```

Comment:
These could be combined into a group by query (see [the docs](http://docs.sqlalchemy.org/en/rel_1_0/orm/tutorial.html#counting))
```python
db.session.query(DraftService.status, func.count(DraftService.status)).group_by(DraftService.status)
```
DraftService.query.